src/components/Config/Config.tsx

```diff
@@ -8,9 +8,28 @@ import { capitalizeFirstLetter } from '../../utils';
 import CopyInput from '../CopyInput';
 import PageTitle from '../PageTitle';
 import ConnectionSettings from './ConnectionSettings';
+import { detect } from 'detect-browser';
 
 const { Link, Text } = Typography;
 
+function FirefoxAlert() {
+  const browser = detect();
```

Comment:
Please move this outside of the `FirefoxAlert` function, to e.g. line 14. The reason is that when it's inside the function then it will run each time the component renders (which can be many, many times) and this type of detection has to run just once (you don't expect the user's browser to suddenly change between renders).